dojo/settings/settings.dist.py

```diff
@@ -1250,6 +1250,7 @@ def saml2_attrib_map_format(dict):
     'Whispers': ['vuln_id_from_tool', 'file_path', 'line'],
     'Blackduck Hub Scan': ['title', 'vulnerability_ids', 'component_name', 'component_version'],
     'BlackDuck API': ['unique_id_from_tool'],
+    'Blackduck Binary Analysis': ['title', 'file_path', 'vuln_id_from_tool', 'unique_id_from_tool'],
```

Comment:
```suggestion
    'Blackduck Binary Analysis': ['title', 'file_path', 'vuln_id_from_tool'],
```

Reply:
Removed completely in favor of relying upon updated `unique_id_from_tool`.